### ISSUE TITLE
[YouTube Downloader] ...

### ISSUE THREAD

**Issue by p-saovapakhiran** [raycast/extensions#11822]
### Extension

https://www.raycast.com/vimtor/youtube-downloader

### Description

"Output format mp4 is not available" error from "YouTube Downloader" when trying to download a YouTube video

### Steps To Reproduce

1. Toggle Raycast
2. Run "YouTube Downloader"
3. Paste a YouTube video URL
4. Click "Download"
5. Download starts but gets "Output format mp4 is not available" or "Something went wrong" error at the end

<img width="862" alt="Screenshot 2024-04-16 at 5 18 02 PM" src="https://github.com/raycast/extensions/assets/3438448/7bc76cf1-594a-441d-8608-ee09118bcc7b">

**Comment by vimtor** [comment]
Hey @p-saovapakhiran! I'm sorry you are facing this issue.

I believe in this case the problem is that this particular video is no longer available. Could you try with another video?

**Comment by vimtor** [comment]
Please @p-saovapakhiran, find the instructions to temporarily solve the issue in the mentioned issue.

**Comment by p-saovapakhiran** [comment]
Hi @vimtor, thank you so much for your prompt response! Unfortunately, I still get the error below:

Error: Output format mp4 is not available

capabilities.js:589:20

---
586:           }, []);
587: 
588:         if (unavailable.length === 1) {
589:           return cb(new Error('Output format ' + unavailable[0] + ' is not available'));
590:         } else if (unavailable.length > 1) {
591:           return cb(new Error('Output formats ' + unavailable.join(', ') + ' are not available'));
592:         }
---

T:index.js:46:13445
I:index.js:46:13524
index.js:43:11429

I installed ffmpeg@6 and checked the paths to ffmpeg and ffprobe if they are correct.
I also tried different YouTube videos including public ones, but none of them worked.
Also tried uninstalling and reinstalling the extension but it also didn't work.

**Comment by vimtor** [comment]
@p-saovapakhiran could you please share the output of running `ffmpeg -version` on your terminal? You may have to uninstall the previous version and link the new one.

**Comment by p-saovapakhiran** [comment]
Hi @vimtor, it turned out I was still using v7. I uninstalled ffmpeg and then install ffmpeg@6 and then ran `brew link ffmpeg@6` and now it's working!

Thank you for your help again and I'm sorry I didn't check it carefully.

**Comment by LexDeBash** [comment]
I solved the problem as follows: First, I opened the terminal and uninstalled the current version of FFmpeg by running brew uninstall ffmpeg. Then, I installed version 6 of FFmpeg using the command brew install ffmpeg@6. After the installation was complete, I linked this version to ensure it was recognized by the system, using the command brew link ffmpeg@6 --force. Following all these steps, I reinstalled the extension. Now everything is working properly!
